.storybook/Header/styles.css

```diff
@@ -0,0 +1,10 @@
+:local .Header {
```

Comment:
Why is `.Header` is title-case but the others are camel-case (e.g. `.swatchColour`)?